actions/playerMatchesActions.js

```diff
@@ -0,0 +1,81 @@
+import fetch from 'isomorphic-fetch';
+import { HOST_URL } from './';
+
+const url = (playerId, numMatches) => `/api/players/${playerId}/matches?limit=${numMatches}`;
```

Comment:
For future consideration--the filters are also all implemented as query params.  Start thinking about how we can work that in.